packages/vite-plugin-cloudflare/playground/dev-vars/package.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "@playground/dev-vars",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"check:types": "tsc --build",
+		"dev": "vite dev",
+		"preview": "vite preview"
+	},
+	"devDependencies": {
+		"@cloudflare/vite-plugin": "workspace:*",
+		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@cloudflare/workers-types": "^4.20241230.0",
+		"typescript": "catalog:vite-plugin",
```

Comment:
Note, mostly unrelated to this PR:

The `vite-plugin` catalog pins `@types/node` but that will not work as expected because of

```json
		"overrides": {
			// ...      
			"@types/node": "$@types/node"
		},
```

So there would need to be a more specific override there for the plugin package